### PR TITLE
feat: import-pact-file (v2/v3/v4 http)

### LINKED ISSUE #10: Import pacts files as Spaces
It would be cool to import pact files, which adhere to the [Pact Specification](https://github.com/pact-foundation/pact-specification), into Explore Spaces.

This would allow users to take a Pact contract, and replay it against a potentially live service, which can aid in reusing contracts for exploration purposes.

There is a similar [tool](https://github.com/ITV/pmpact) which provides Pact -> Postman 

There are json schemas available for each of the 4 Pact specifications

https://github.com/pactflow/pact-schemas

### PR DESCRIPTION
**Pull request by YOU54F**
fixes #10 

- Validates the following pact formats utilising https://github.com/pactflow/pact-schemas
  - V1
  - V2
  - V3
  - V4
  - ⚠️ ~~currently possible incorrect warnings about valid matchers. Can remove matchers from schema and perform upstream changes later to https://github.com/pactflow/pact-schemas~~
    - Have now set matchers, matchingRules, generators to be allowed as property keys but provide no validation, and they are removed from the object mapper.
- Processes the following interaction types
  - V2
  - V3 sync/http
  - V3 sync/http
  - V4 sync/http
  - ⚠️ ~~TODO - explicitly detect and ignore incompatible types~~
    - Will filter out non `"Synchronous/HTTP"` interaction types for v4 pacts.
- Creates new scoped option `--base-uri` / `-b` for user to provide url override when uploading pacts as Pact's do not contain that info
- Creates new scoped option `--ignore-pact-schema-verification-result` for user to ignore the validation performed against the pact-schemas. The files may, or may not be supported if they fail validation, but allows an escape hatch, just in case it is an issue on our side to fix with the schemas

### CODE REVIEW

**Comment by YOU54F** [comment]
Could do with dry-ing up, but good enough for a review and able to upload all the new pact fixtures to explore

File: README.md [hunk 1]
```diff
@@ -295,6 +297,47 @@ From SwaggerHub Explore, navigate to your browser development tools, locate the
...
+  > `-v`, `--verbose`                                      Include verbose output during processing
+
+  > `-?`, `-h`, `--help`                                   Show help and usage information
+
+**Note** - the format for SwaggerHub Explore cookies is as follows: `"cookie-name=cookie-value; cookie-name=cookie-value"`
+
+>Example: `"SESSION=5a0a2e2f-97c6-4405-b72a-299fa8ce07c8; XSRF-TOKEN=3310cb20-2ec1-4655-b1e3-4ab76a2ac2c8"`
+
+> **Notes:**
+> - Compatible with Valid Pact v2 / v3 / v4 specification files
```
Comment (frankkilcommins) [inline README.md:333]:
```suggestion
> - Compatible with Valid Pact v2 / v3 / v4 specification files
```
```suggestion
> - Compatible with valid Pact v2 / v3 / v4 specification files
```

**Review by frankkilcommins — APPROVED** [review]
LGTM - and some of this will already be refactored with adding Pact tests.

File: README.md (hunk 1 above)
Comment (YOU54F) [reply README.md]: done, ta

**Pull request merged**